index.html

```diff
@@ -2418,14 +2423,22 @@
             </li>
             <li>Add <var>S</var> to the <a>set of presentation controllers</a>.
             </li>
+            <li>If there is an unsettled <a>Promise</a> <var>P</var> from a
+            previous call to get the <a for=
+            "PresentationReceiver">connectionList</a> attribute for the same
```

Comment:
With the update above, this could be simply "If there is an unsettled Promise P associated with the PresentationReceiver object, ..."